npm-update.js

```diff
@@ -1,189 +1,88 @@
 /* Copyright (c) 2014-2015 Richard Rodger, MIT License */
 /* jshint node:true, asi:true, eqnull:true */
-"use strict";
+"use strict"
 
 
-var follow = require('follow');
-var npm_stats = require('npm-stats')
-var async     = require('async')
+var follow     = require('follow')
+var NpmStats   = require('npm-stats')
+var async      = require('async')
+var JSONStream = require('JSONStream')
 
 
-module.exports = function npm_update( options ){
+module.exports = function npmUpdate( options ){
   var seneca = this
 
-  options = seneca.util.deepextend({
-    task: null,
-    batchsize: 33,
+  var opts = Object.assign({
```

Comment:
Strictly for consistency. Underneath it is using lodash. Once seneca drops support for 0.x We should probably deprecate this whole recipe. The only reason for us using it is our code is 'demo' code too. Other es6 features are fine though.